tauri-app/src-tauri/src/seventh_heaven.rs

```diff
@@ -66,7 +90,15 @@ pub(crate) async fn install_run(app_handle: AppHandle) -> Result<(), ()> {
     }).unwrap();
 
     with_status(&app_handle,"Setting up FF7...".to_string(), || -> io::Result<()> {
-        todo!("Fetch FF7 and install it");
+        let game_path = steam.get_game_path(39140);
+        if !SteamManager::can_read_path(&game_path) {
+            panic!("We can't read the game path at {:?}", game_path)
+        }
+        match copy_directory(game_path.as_path(), &wine_manager.get_c_path("FF7")) {
+            Ok(_) => Ok(println!("We copied to {:?} successfully!",
+                &wine_manager.get_c_path("FF7"))),
+            Err(err) => panic!("Nooooo! {err}")
```

Comment:
While not intuitive, you should propagate the error, so that `with_status` can handle it

Reply:
Can you please show me how to do this? Maybe rewrite this one for me so I can fix the others?

I did already beat my head against the wall for an hour or so trying to figure out how to properly send errors to the GUI but I had no success so I pushed this in the end.